projectile.el

```diff
@@ -1292,7 +1293,8 @@ If ignored directory prefixed with '*', then ignore all
 directories/subdirectories with matching filename,
 otherwise operates relative to project root."
   (let ((ignored-files (projectile-ignored-files-rel))
-        (ignored-dirs (projectile-ignored-directories-rel)))
+        (ignored-dirs (projectile-ignored-directories-rel))
+        (filtering-regexes (mapcar #'eshell-glob-regexp (car (projectile-filtering-patterns)))))
```

Comment:
This bit of code puzzles me - why take just the `car` of the filtering patterns?